# Essa é a mensagem que de fato é enviada por uma estratégia para o VSS-Viewer

# Lista dinâmica de vetores de movimentação. Sendo sempre 3, devido a quantidade de robôs de um time
d_Pose[] step_poses

# Lista dinâmica de poses futuras. Sendo sempre 3, devido a quantidade de robôs de um time
d_Pose[] final_poses

# Lista dinâmica de caminhos. Snedo sempre 3, devido a quantidade de robôs de um time
d_Path[] paths
